src/gseproject/templates/game/tree.html

```diff
@@ -44,85 +86,73 @@
-        } else {
-            clearInterval(interval); // Stop interval when tree is fully grown
-            cherries += 10; // Bonus cherries for growing a tree
-            alert('Congratulations! You have grown a full tree and earned 10 bonus cherries!');
-            plantingInProgress = false; // Reset planting flag
-            checkCooldown(); // Check cooldown for next planting
-        }
-    }, 2000); // Change stage every 2 seconds
+            document.getElementById('treeImage' + (currentStage)).style.display = 'block';
+            console.log('Current stage(1,2,3,4):', currentStage); // Log the current stage
+            if(currentStage === stages.length){
+                console.log('Tree is fully grown!'+currentStage); // Log when tree is fully grown
+                
+                clearInterval(interval); // Stop interval when tree is fully grown  
+                plantingInProgress = false; // Reset planting flag
+            }
+            
+        } 
+    }, 10000); // Change stage every (10) seconds for testing
 }
 
-function updateCherriesDisplay() {
-  document.getElementById('cherriesCount').innerText = `Cherries: ${cherries}`;
-}
 
-function checkCooldown() {
-    let lastPlantedTime = localStorage.getItem('lastPlantedTime');
-    if (lastPlantedTime) {
-        let timePassed = new Date().getTime() - lastPlantedTime;
-        if (timePassed < cooldownDuration) {
-            // Disable plant button during cooldown
-            document.getElementById('plantButton').disabled = true;
-            // Start countdown timer
-            countdownTimer((cooldownDuration - timePassed) / 1000);
-        } else {
-            // If cooldown ended, enable plant button
-            document.getElementById('plantButton').disabled = false;
-        }
-    }
-}
 
 function countdownTimer(seconds) {
     // Display countdown timer
     let timerElement = document.getElementById('countdownTimer');
-    timerElement.textContent = `Cooldown ends in: ${formatTime(seconds)}`;
+    timerElement.textContent = `Countdown ends in: ${formatTime(seconds)}`;
     
     // Update timer every second
     let countdown = setInterval(() => {
         seconds--;
-        timerElement.textContent = `Cooldown ends in: ${formatTime(seconds)}`;
+        timerElement.textContent = `Countdown ends in: ${formatTime(seconds)}`;
         if (seconds <= 0) {
             clearInterval(countdown); // Stop countdown when timer reaches zero
             document.getElementById('plantButton').disabled = false; // Enable plant button
             timerElement.textContent = ''; // Clear timer display
+            if (currentStage === stages.length) {
+                incrementCherriesValue(); // Increment cherries value when tree is fully grown
+                alert('Congratulations! You have grown a full tree and earned 10 bonus cherries!, carbon footprint reduced by 1!');
```

Comment:
It would be nicer if this text apperead in a pop up on the page, like some boostrap conatiner rather than an alert

Reply:
i understand this might be alot, so feel free to leave if it is, if you could just have a very quick look into it

Reply:
hmmm, i could do that in the future, but it is good for now because im really exhausted lul

Reply:
thats fine